.goreleaser.yml

```diff
@@ -52,6 +52,34 @@ before:
         ".releaseSeries |= (. + [{contract: \"v1beta1\", major: {{ .Major }}, minor: {{ .Minor }}}] | unique)" \
         metadata.yaml >release-metadata.yaml'
     - make template-helm-repository
+    # dev image
+    - |
+      sh -ec 'if [ {{ .IsSnapshot }} == true ] ; then
+        GOOS=linux \
+        SOURCE_DATE_EPOCH=$(date +%s) \
+        KO_DOCKER_REPO=ko.local/{{ .ProjectName }} \
+          ko build \
+          --bare \
+          --tags v{{ trimprefix .Version "v" }} \
+          ./cmd
+      fi'
+    # released image
+    - |
+      sh -ec 'if [ {{ .IsSnapshot }} == false ] ; then
```

Comment:
Moving it here means that the images are pushed before the artifacts are uploaded to Github. I think thats ok though since its already possible for one of these steps to fail and have a half broken release until its rerun.